docs/src/languages/java.md

```diff
@@ -40,6 +44,8 @@ We recommend you install one or the other and not both.
 
 ## Settings / Initialization Options
 
+For available `initialization_options` please see the [Initialize Request section of the Eclipse.jdt.ls Wiki](https://github.com/eclipse-jdtls/eclipse.jdt.ls/wiki/Running-the-JAVA-LS-server-from-the-command-line#initialize-request).
+
 See [JDTLS Language Server Settings & Capabilities](https://github.com/eclipse-jdtls/eclipse.jdt.ls/wiki/Language-Server-Settings-&-Capabilities) for a complete list of options.
```

Comment:
Like I said earlier, this page is mostly related to compilation options and not initialization options, so this line should be removed.

Reply:
Thanks. I'm two decades out from the programming full time and so this is all very theoretical for me. :P